### DISCUSSION TITLE
Attribution Model for v1.2

### DISCUSSION THREAD

**Discussion by ghost** [USEPA/supply-chain-factors#64] [Q&A] [answered]
There are links to the v1.1.1. model that allow users to download the underlying National GHG attribution model. The links provided in the v1.2 release seem to link to the same underlying attribution model. Is there a new National GHG attribution model for v1.2? If so when will it be available?

**Comment by bl-young** [comment, marked as answer]
The documentation for the v.1.2 update can be found [here](https://pasteur.epa.gov/uploads/10.23719/1528686/documents/Aboutthe2019v1.2SupplyChainGHGEmissionFactors.pdf). For this update, the GHG modeling was performed in EPA's [FLOWSA tool](https://www.mdpi.com/2076-3417/12/11/5742), which is a python tool meant to replace the original National GHG attribution model in Excel. The core method file for the v1.2. release can be found in FLOWSA v1.3.1 [here](https://github.com/USEPA/flowsa/blob/v1.3.1/flowsa/methods/flowbysectormethods/GHG_national_m1.yaml). We are currently working on more extensive documentation for the GHG modeling done in FLOWSA
